utils/python/CIME/XML/generic_xml.py

```diff
@@ -28,7 +30,10 @@ def __init__(self, infile=None):
         else:
             # if file does not exist create a root xml element
             # and set it's id to file
+
+            logger.warning("File %s does not exists." , infile)
```

Comment:
Should be a warning so if the file is missing a user knows that the case dir is incomplete and something is broken?